### PR TITLE
Docs csi 4159 release 1.9 2nd ed rn

### PR DESCRIPTION
**Pull request by rivkap**
2nd edition release notes for 1.7.0.
Moved CSI-702 from Known issues to Fixed issue change log.

### CODE REVIEW

File: docs/content/release_notes/changelog_1.7.0.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # 1.7.0 (September 2021)
 
-IBM® block storage CSI driver 1.7.0 added new support and enhancements:
+IBM® block storage CSI driver 1.7.0 added new support and enhancements and resolved the following issue:
 - Now supports the CSI Topology feature
```
Comment (oriyarde) [inline docs/content/release_notes/changelog_1.7.0.md:4]: isn't it weird to end the sentence with "the following issue" and then start with the new support and enhancements?

Comment (rivkap) [reply docs/content/release_notes/changelog_1.7.0.md:4]: updated wording in all three prs

Comment (oriyarde) [reply docs/content/release_notes/changelog_1.7.0.md:4]: I don't see the change in this PR specifically

Comment (rivkap) [reply docs/content/release_notes/changelog_1.7.0.md]: please check again. i had forgotten to push my cange earlier.

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Pull request merged**